### PR TITLE
fix: Correct minor spelling mistakes

### PR DESCRIPTION
**Pull request by tartansandal**

### CODE REVIEW

**Review by goncalossilva — APPROVED** [review]

**Comment by tartansandal** [comment]
@goncalossilva Just to note, I don't have write access to this repo or to https://github.com/Doist/fixedlist so I can't merge any PRs :cry:

**Comment by goncalossilva** [comment]
@tartansandal the backend team wasn't added to either. 😅 Corrected that, you should be able to merge now!

**Pull request merged**
